ReadMe.txt

```diff
@@ -0,0 +1,29 @@
+# purpose of a project
+The project aims to create a simple and convenient tool for converting scripts written in Python, in a sequence of shell commands bash.
```

Comment:
The project aims to create a simple and convenient tool for converting scripts written in Python, in a sequence of shell commands bash. - > The project aims to create a simple and convenient tool for converting scripts written in Python, in a sequence of bash commands.